__mocks__/react-instantsearch-dom.jsx

```diff
@@ -31,6 +31,7 @@ const fakeHits = [
       start: mockCurrentStartDate,
       end: dayjs(mockCurrentStartDate).add(1, 'year').toISOString(),
       enroll_by: dayjs(mockCurrentStartDate).unix(),
```

Comment:
nit: curious whether we can either rename this mock date to be more generic or have separate variables to use for each related field.

As is, the `mockCurrentStartDate` is being used for `end`/`enroll_by` dates, too which could be a bit misleading/confusing.